core/src/main/java/com/yandex/yoctodb/v1/mutable/V1DatabaseBuilder.java

```diff
@@ -157,7 +158,12 @@ public void writeTo(
                     final OutputStream os) throws IOException {
                 // Header
                 os.write(DatabaseFormat.MAGIC);
-                os.write(Ints.toByteArray(V1DatabaseFormat.FORMAT));
+                os.write(Ints.toByteArray(
+                    Feature.intValue(
+                        Feature.LEGACY,
```

Comment:
I don't quite understand that -- we are expected to signal **used** features, not supported ones in the persistent format. How are we supposed to drop `LEGACY` support with your approach?

Reply:
I assumed that LEGACY would indicate the previous version (6), but now I think you're right. Dropped LEGACY from the builder.